testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/messaging/security/DeserializationBlockListTestCase.java

```diff
@@ -96,7 +96,7 @@ public static JavaArchive createArchive() {
     private DeserializationMessagingBean bean;
 
     @Test
-    public void testDeserializationBlackList() throws NamingException {
+    public void testDeserializationBlockList() throws NamingException {
         // UUID is black listed, any other Serializable must be deserialized.
```

Comment:
Hi @parsharma , would you mind adding a new commit to fixing this comment too?

Reply:
Done https://github.com/wildfly/wildfly/pull/15443